.circleci/publish-git-tag.sh

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env bash
```

Comment:
> When you're not using bash-specific instructions, `sh` is safer: `bash` is an implementation with extensions while `sh` is an interface.

Reply:
I'd add that _usually_ `sh` is just a pointer to `bash`, but *not always*.